src/Swarm/TUI/Controller.hs

```diff
@@ -272,7 +272,7 @@ stripCmd pty                    = pty
 handleREPLEvent :: AppState -> BrickEvent Name AppEvent -> EventM Name (Next AppState)
 handleREPLEvent s (VtyEvent (V.EvKey (V.KChar 'c') [V.MCtrl]))
   = continue $ s
-      & gameState . robotMap . ix "base" . machine .~ idleMachine
+      & uiState . uiReplForm %~ updateFormState ""
```

Comment:
Not so familiar with that part of the code, @byorgey should confirm, but I think we need to keep the idleMachine assignment for the base, e.g.:

 ```suggestion
       & (uiState . uiReplForm %~ updateFormState "")
        . (gameState . robotMap . ix "base" . machine .~ idleMachine)
```

Reply:
Yes, the point of the `idleMachine` assignment is that if the base is in the middle of running a computation but you want it to stop, you can hit Ctrl-C and it will replace its currently running machine with a machine that does nothing.

I am not sure off the top of my head why three dots are displayed when you type something at the REPL and then hit Ctrl-C.  The three dots are displayed when the base is actively running a program.  My guess is that it is "actively running" the `idleMachine` for a tick or two and then realizes that there is nothing to do.  Ideally, no dots would be displayed at all when you Ctrl-C.

Reply:
I see, yes, it is because of the way `idleMachine` is defined: it has to actually execute a `Noop` instruction, but this is silly.  Instead we should just define `idleMachine = Out VUnit []`.  @mateoatr , if you want to change that definition as part of this PR that would be great, or we could do it separately.

Reply:
Hmm, I think we also want to explicitly set `gameState . replStatus .~ REPLDone`, so the `replStatus` doesn't get out of sync with what the base is actually doing.